semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/SymbolOps.scala

```diff
@@ -8,34 +8,35 @@ import scala.meta.internal.{semanticdb3 => s}
 import scala.meta.internal.semanticdb3.Scala._
 import scala.meta.internal.semanticdb3.Scala.{Descriptor => d}
 import scala.util.control.NonFatal
+import scala.meta.internal.semanticdb3.Scala._
 
 trait SymbolOps { self: SemanticdbOps =>
 
-  lazy val symbolCache = new HashMap[g.Symbol, m.Symbol]
+  lazy val symbolCache = new HashMap[g.Symbol, String]
   implicit class XtensionGSymbolMSymbol(sym: g.Symbol) {
-    def toSemantic: m.Symbol = {
-      def uncached(sym: g.Symbol): m.Symbol = {
-        if (sym == null || sym == g.NoSymbol) return m.Symbol.None
-        if (sym.isOverloaded) return m.Symbol.Multi(sym.alternatives.map(_.toSemantic))
+    def toSemantic: String = {
+      def uncached(sym: g.Symbol): String = {
+        if (sym == null || sym == g.NoSymbol) return Symbols.None
+        if (sym.isOverloaded) return Symbols.Multi(sym.alternatives.map(_.toSemantic))
         if (sym.isModuleClass) return sym.asClass.module.toSemantic
         if (sym.isTypeSkolem) return sym.deSkolemize.toSemantic
         if (sym.isSemanticdbLocal) return freshSymbol(sym.pos.toMeta.input)
 
         val owner = sym.owner.toSemantic
         val signature = {
           if (sym.isMethod || sym.isUsefulField) {
-            m.Signature.Method(sym.name.toSemantic, sym.disambiguator)
+            Descriptor.Method(sym.name.toSemantic, sym.disambiguator)
```

Comment:
The convention that I've been used in other parts of Scalameta is to say `d.Method`, `d.TypeParameter`, etc. There is already `import scala.meta.internal.semanticdb3.Scala.{Descriptor => d}` in scope in this file.

Reply:
Fixed.